kolibri/plugins/learn/assets/src/vue/breadcrumbs/index.vue

```diff
@@ -44,23 +98,58 @@
         };
       },
     },
+    vuex: {
+      getters: {
+        pageMode: getters.pageMode,
+        crumbs: state => (state.pageState.topic ? state.pageState.topic.breadcrumbs : null),
+        isRoot: state => (state.pageState.topic ?
+          state.pageState.topic.id === state.rootTopicId : false),
+        pageName: state => state.pageName,
+        pageState: state => state.pageState,
+      },
+    },
   };
 
 </script>
 
 
 <style lang="stylus" scoped>
 
+  @require '~core-theme.styl'
+  @require '../learn.styl'
+
+  .sep
+    margin-left: 0.5em
+    margin-right: 0.5em
   .nav
     margin-top: 2em
     margin-bottom:1.4em
-
-  .parent a:link
-    font-weight: 300
-
-  span.parent::after
-    content: '»'
+  .parent:after
+    content: '>'
     margin-left: 0.5em
     margin-right: 0.5em
+    color: $core-text-annotation
+  a
+    display: inline-block
+    vertical-align: middle
+    margin-bottom: 2px
+    font-size: 0.9em
+    font-weight: 300
+    max-width: 140px
+    white-space: nowrap
+    overflow: hidden
+    text-overflow: ellipsis
+    color: $core-text-annotation
+  svg
+    vertical-align: middle
+    fill: $core-text-annotation
+    margin-bottom: 2px
+  .full-breadcrumbs
+    @media screen and (max-width: $portrait-breakpoint)
+      display: none
+  .back
+    display: none
+    @media screen and (max-width: $portrait-breakpoint)
+      display: initial
```

Comment:
please add endlines between classes for consistency with the rest of our code base